subiquity/models/filesystem.py

```diff
@@ -846,8 +846,8 @@ def _available(self):
 
 @fsobj("mount")
 class Mount:
-    device = attributes.ref(backlink="_mount")  # Filesystem
     path = attr.ib()
+    device = attributes.ref(backlink="_mount", default=None)  # Filesystem
```

Comment:
Is the distinction between None and NOTHING intentional?
If not we can omit this change.

Reply:
yes, default=attr.NOTHING means there is no default, so the attribute must be supplied when the instance is constructed, default=None means the attribute defaults to None.